### PR TITLE
Add dayjs and update README

### PR DESCRIPTION
**Pull request by ddvkid**
add comparison by dayjs
#6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,6 +160,8 @@ moment("2010-10-20 4:30", "YYYY-MM-DD HH:mm");
 import parse from "date-fns/parse";
 parse("2010-10-20 4:30", "yyyy-MM-dd H:mm", new Date());
 // => "2010-10-19T17:30:00.000Z"
+
+// dayjs ❌ not support custom format parse
```
Comment (stevemao) [inline README.md]: I don't think you need to mention `dayjs` if it's not supported

Comment (ddvkid) [reply README.md]: I think this comparison would help people make a decision when choosing date lib to replace moment.js.

Comment (cht8687) [reply README.md]: `not support` => `do not support`

File: README.md [hunk 2]
```diff
@@ -322,6 +407,8 @@ moment().isoWeeksInYear();
 import getISOWeeksInYear from "date-fns/getISOWeeksInYear";
 getISOWeeksInYear(new Date());
 // => 52
+
+// dayjs ❌ not support weeks in year
```
Comment (cht8687) [inline README.md]: in the year?

File: README.md [hunk 3]
```diff
@@ -345,6 +432,8 @@ moment.max(array.map(a => moment(a)));
 import max from "date-fns/max";
 max(array);
 // => "2018-03-11T13:00:00.000Z"
+
+// dayjs ❌ not support maximum of the given dates
```
Comment (cht8687) [inline README.md]: the maximum?

**Comment by stevemao** [comment]
I think we should also give user some information on which one to choose for different use cases. Benchmark would be very useful. Users should be able to cherry-pick different modules in the same project thanks to tree shaking (if not, state the reason).

**Comment by ddvkid** [comment]
@stevemao  I think the benchmark is a good point, let's create a ticket for it.

File: __tests__/index.js [hunk 4]
```diff
@@ -1,14 +1,25 @@
...
+dayjs.extend(IsBetween);
+dayjs.extend(IsLeapYear);
 
 const time = 1536484369695;
 
 describe("Parse", () => {
   it("String + Date Format", () => {
     const m = moment("12-25-1995", "MM-DD-YYYY");
     const d = date.parse("12-25-1995", "MM-dd-yyyy", new Date());
+    const day = dayjs("12-25-1995");
```
Comment (cht8687) [inline __tests__/index.js]: make some naming convention here? like `date_fns_day` and `dayjs_day` to make it easier for other libs

File: README.md [hunk 5]
```diff
@@ -188,6 +204,20 @@ new Date().getSeconds();
 // => 49
 new Date().getHours();
 // => 19
+
+// date-fns
+import getSeconds from "date-fns/getSeconds";
+import getHours from "date-fns/getHours";
+getSeconds(new Date());
```
Comment (stevemao) [inline README.md]: I'd like to know why using this is better than the native method.

File: __tests__/index.js [hunk 6]
```diff
@@ -25,56 +36,96 @@ describe("Parse", () => {
 describe("Get + Set", () => {
   it("get Second", () => {
     const m = moment(time).seconds();
-    const d = new Date(time).getSeconds();
+    const n = new Date(time).getSeconds();
+    const d = date.getSeconds(new Date(time));
+    const day = dayjs(time).second();
+    expect(m).toBe(n);
```
Comment (cht8687) [inline __tests__/index.js]: same naming suggestions applied to all below logic

File: README.md (hunk 5 above)
Comment (ddvkid) [reply README.md]: it's just a comparison, do we need to give the best one we think? or just let people select what they want.

File: README.md (hunk 1 above)
Comment (stevemao) [reply README.md]: @ddvkid then we should list the pros and cons of each lib.

**Comment by stevemao** [comment]
@ddvkid https://github.com/you-dont-need/You-Dont-Need-Momentjs/issues/3

File: README.md (hunk 5 above)
Comment (stevemao) [reply README.md]: I think we should list the pros and cons.

File: README.md [hunk 7]
```diff
@@ -283,11 +357,18 @@ moment().week(24);
...
 // date-fns
 import getWeek from "date-fns/getWeek";
+import setWeek from "date-fns/setWeek";
 getWeek(new Date());
 // => 37
-import setWeek from "date-fns/setWeek";
 setWeek(new Date(), 24);
 // => "2018-06-10T09:12:49.695Z"
+
+// dayjs ⚠️ requires plugin
```
Comment (stevemao) [inline README.md]: dayjs ⚠️ requires weekOfYear plugin

**Comment by stevemao** [comment]
I think screenshot.png is a nice image. Why is it removed?

File: README.md (hunk 5 above)
Comment (stevemao) [reply README.md]: Otherwise, people will ask "why not X lib/module?"

**Pull request merged**